### PR TITLE
HDDS-4354. Avoid spamming of logs because of deleted transactions.

### PR DESCRIPTION
**Pull request by mukul1987**
## What changes were proposed in this pull request?

Avoid spamming of logs because of deleted transactions.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-4354

## How was this patch tested?
MiniOzoneChaosCluster

### CODE REVIEW

**Review by ayushtkn — APPROVED** [review]

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java [hunk 1]
```diff
@@ -130,9 +130,7 @@ public void incrementCount(List<Long> txIDs) throws IOException {
         DeletedBlocksTransaction block =
             scmMetadataStore.getDeletedBlocksTXTable().get(txID);
         if (block == null) {
-          // Should we make this an error ? How can we not find the deleted
-          // TXID?
-          LOG.warn("Deleted TXID {} not found.", txID);
+          LOG.debug("Deleted TXID {} not found.", txID);
```
Comment (lokeshj1703) [inline hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java:133]: NIT: We can include the log inside check for debug log enabled.
```suggestion
          if (LOG.isDebugEnabled()) {
            LOG.debug("Deleted TXID {} not found.", txID);
          }
```

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lease/LeaseManager.java [hunk 2]
```diff
@@ -235,7 +235,7 @@ public void run() {
           }
         } catch (InterruptedException e) {
           // This means a new lease is added to activeLeases.
-          LOG.error("Execution was interrupted ", e);
+          LOG.debug("Execution was interrupted ", e);
```
Comment (lokeshj1703) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/lease/LeaseManager.java:238]: This should not be related to block deletion. What exception are we seeing here?

**Review by lokeshj1703 — COMMENTED** [review]
@mukul1987  Thanks for working on this! I have minor comments inline.

**Comment by elek** [comment]
/pending Pending on the minor comments from @lokeshj1703

**Pull request closed without merging**